client/js/views/templates/AddOffering.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function({ model }) {
+	return `<div>
+	<form>
+	  Item Name:<br>
```

Comment:
use `<div>` instead of `<br>` here:

```
<form>
    <div>Item Name:</div>
    <div><input data-js="name" type="text"></div>
```

It makes styling with classes easier.